infra/scripts/xania.logrotate

```diff
@@ -2,5 +2,7 @@
 /home/xania/data/log/doorman.log
 {
     daily
+    size 1M
```

Comment:
@mattgodbolt  should be a simple fix for the 'zero byte log file' issue. It'll also cut down on redundant rotation where log files are really small. 

As a side note, the `append:` mode we set in the service configs for `StandardOutput` only works with systemd >= 240, and by default 18.04 only ships with 237, which only supports `file:`. Discovered while trying this stuff out locally.

Reply:
To deploy this I'll end up copying this file into the right place, what the init scripts do.

Reply:
Sounds perfect.